src/compiler/checker.ts

```diff
@@ -7555,6 +7559,62 @@ namespace ts {
             return undefined;
         }
 
+        function shouldAcquireLiteralType(literalNode: StringLiteral) {
+

```

Comment:
The whitespace here and below is intentional and it's for readability so that everything doesn't look like one big clump of code.